ui/src/overview.tsx

```diff
@@ -50,17 +55,77 @@ import {
   getCriteriaPlugin,
 } from "./cohort";
 
+const drawerWidth = 400;
+
+// This drawer customization is taken directly from the MUI docs.
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create("width", {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  }),
+  overflowX: "hidden",
+});
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create("width", {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  overflowX: "hidden",
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up("sm")]: {
+    width: `calc(${theme.spacing(8)} + 1px)`,
+  },
+});
+
+const Drawer = styled(MuiDrawer, {
+  shouldForwardProp: (prop) => prop !== "open",
+})(({ theme, open }) => ({
+  width: drawerWidth,
+  flexShrink: 0,
+  whiteSpace: "nowrap",
+  boxSizing: "border-box",
+  ...(open && {
+    ...openedMixin(theme),
+    "& .MuiDrawer-paper": openedMixin(theme),
+  }),
+  ...(!open && {
+    ...closedMixin(theme),
+    "& .MuiDrawer-paper": closedMixin(theme),
+  }),
+}));
+
 export default function Overview() {
   const cohort = useCohort();
 
+  const [showDemographics, setShowDemographics] = useState(false);
+
   return (
     <>
       <ActionBar title={cohort.name} />
-      <Grid container columns={3} columnSpacing={5} className="overview">
-        <ParticipantsSelector kind={tanagra.GroupKindEnum.Included} />
-        <ParticipantsSelector kind={tanagra.GroupKindEnum.Excluded} />
-        <DemographicCharts cohort={cohort} />
-      </Grid>
+      <Box sx={{ display: "flex" }}>
+        <Box sx={{ flexGrow: 1 }}>
+          <Grid container columns={2} columnSpacing={5} className="overview">
+            <ParticipantsSelector kind={tanagra.GroupKindEnum.Included} />
+            <ParticipantsSelector kind={tanagra.GroupKindEnum.Excluded} />
+          </Grid>
+        </Box>
+        <Drawer variant="permanent" open={showDemographics} anchor="right">
+          <Box sx={{ m: 1 }}>
+            <Toolbar />
+            <Box sx={{ float: "right" }}>
+              <IconButton
+                size="large"
+                onClick={() => setShowDemographics(!showDemographics)}
+              >
+                {showDemographics ? <ChevronRightIcon /> : <BarChartIcon />}
+              </IconButton>
+            </Box>
+            <DemographicCharts cohort={cohort} open={showDemographics} />
```

Comment:
{showDemographics && <DemographicsChar cohort={cohort} />}

instead of passing in the prop to return null